cmd/agent/app/options/options.go

```diff
@@ -78,7 +78,8 @@ type GrpcProxyAgentOptions struct {
 	// The check is an "unlocked" read but is still use at your own peril.
 	WarnOnChannelLimit bool
 
-	SyncForever bool
+	SyncForever    bool
+	XfrChannelSize uint
```

Comment:
Would it be easier to make this an int to avoid casting and possible overflow errors?

Reply:
I have changes to uint due to the previous comment, but taking overflow errors into account, I will use int again and validate against negative and zero values.